packages/adapter-drizzle/src/lib/mysql.ts

```diff
@@ -266,18 +254,193 @@ export function MySqlDrizzleAdapter(
   }
 }
 
-export type MySqlTableFn<T extends TableConfig> = MySqlTableWithColumns<{
-  name: T["name"]
-  columns: T["columns"]
-  dialect: T["dialect"]
+type DefaultMyqlColumn<
+  T extends {
+    data: string | number | Date
+    dataType: "string" | "number" | "date"
+    notNull: boolean
+    columnType: "MySqlVarChar" | "MySqlText" | "MySqlTimestamp" | "MySqlInt"
+  },
+> = MySqlColumn<{
+  name: string
+  columnType: T["columnType"]
+  data: T["data"]
+  driverParam: string | number
+  notNull: T["notNull"]
+  hasDefault: boolean
+  enumValues: any
+  dataType: T["dataType"]
+  tableName: string
+}>
+
+export type DefaultMySqlUsersTable = MySqlTableWithColumns<{
+  name: string
+  columns: {
+    id: DefaultMyqlColumn<{
+      data: string
+      dataType: "string"
+      notNull: true
+      columnType: "MySqlVarChar" | "MySqlText"
```

Comment:
is it required specifying the exact database type here? this is fine for mysql but for example postgres got the UUID datatype that would also work. isn't it enough to specify the `dataType: "string"`?

Reply:
I added uuid type for `columnType`. Unfortunately, `dataType` won't be enough because some `non-string` types with `mode: 'string'` can have the same `dataType` as `string` types.